src/ui/components/Toolbox.tsx

```diff
@@ -6,12 +6,10 @@ import { Redacted } from "./Redacted";
 
 export default function Toolbox() {
   return (
-    <Redacted id="toolbox">
-      <div className="toolbox-top-panels">
```

Comment:
We're still using the `#toolbox` selector in a few CSS spots. We should leave this in, or just consolidate `toolbox`/`toolbox-top-panels`.